scalyr_agent/tests/configuration_k8s_test.py

```diff
@@ -372,3 +375,36 @@ def _test_k8s_ignore_namespaces_supersedes(env_str, local_str, expected):
         _test_k8s_ignore_namespaces_supersedes(None, None, kube_system)
         _test_k8s_ignore_namespaces_supersedes(None, r'""', [])
         _test_k8s_ignore_namespaces_supersedes(r'', r'""', [])
+
+
+class TestK8SUtils(ScalyrTestCase):
+    """
+    Tests for monitor_utils/k8s.py
+    """
+    def setUp(self):
+        super(TestK8SUtils, self).setUp()
+        self.__pod_valid_a = QualifiedName("default", "scalyr-agent-2-75d69db5cc-fcl2s")
+        self.__pod_valid_b = QualifiedName("default", "scalyr-agent-2-75d69db5cc-fvhmw")
+        self.__pod_no_namespace = QualifiedName(None, "scalyr-agent-2-75d69db5cc-fcl2s")
+        self.__pod_no_podname = QualifiedName("default", None)
+        self.__pod_no_namespace_no_podname = QualifiedName(None, None)
+
+    def test_k8s_utils_qualified_name_is_valid(self):
+        assert (self.__pod_valid_a.is_valid())
```

Comment:
Missing this in my comments yesterday, but for consistency with other tests, can you use `self.assertTrue` and `self.assertFalse` for these tests rather than just a plan `assert`. and `assert( not`.